README.md

```diff
@@ -426,6 +444,23 @@ branch.
 - `base_url` (default: `https://api.github.com`) is the base URL to the Github
   API to use (for Github Enterprise support set this to `https://{your_domain}/api/v3`).
 
+### `all_repos.push.bitbucket_server_pull_request`
+
+Pushes the branch to `origin` and then creates a Bitbucket pull request for the branch.
+
+#### Required `push_settings`
+
+- `base_url`: the Bitbucket server URL (eg `bitbucket.domain.com`)
+- `username`: the Bitbucket username you will log in as.
+- `app_password`: the authentication method for the above user to login with
+    - Create an application password within your [account settings](https://bitbucket.domain.com/plugins/servlet/access-tokens/manage).
+    - We need the scope: Repositories -> Read
+
+#### Optional `push_settings`
+
+- `reviewers` (default: `[]`): if set will add these reviewers to the pull request.
+    Example: `['user', 'another.user']`.
```

Comment:
hmmm this seems maybe outside the scope of what to provide out of the box

Reply:
Ok, that part is removed again.